roles/openshift_node/tasks/config/configure-node-settings.yml

```diff
@@ -7,7 +7,7 @@
     create: true
   with_items:
   - regex: '^OPTIONS='
-    line: "OPTIONS=--loglevel={{ openshift.node.debug_level | default(2) }}"
+    line: "OPTIONS=--loglevel={{ openshift.node.debug_level | default(2) }} {{ openshift.node.start_options }}"
```

Comment:
We're trying to move away from openshift_facts so lets forego this. So instead just `{{ openshift_node_start_options | default('') }}`

Reply:
Sounds good. I'll update! And I'm glad we're moving away from the extra level of indirection.